src/vectorchord/usage/search.md

```diff
@@ -0,0 +1,75 @@
+# Search
+
+Get the nearest 5 neighbors to a vector
+
+```sql
+SELECT * FROM items ORDER BY embedding <-> '[3,1,2]' LIMIT 5;
+```
+
+## Operators
+
+These operators are used for distance metrics:
+
+| Name  | Description                |
+| ----- | -------------------------- |
+| `<->` | squared Euclidean distance |
+| `<#>` | negative dot product       |
+| `<=>` | cosine distance            |
+
+## Filter
+
+For a given category, get the nearest 10 neighbors to a vector
+```sql
+SELECT 1 FROM items WHERE category_id = 1 ORDER BY embedding <#> '[0.5,0.5,0.5]' limit 10
+```
+
+## Query options
+
+#### `vchordrq.probes`
+    
+- Description: This GUC parameter `vchordrq.probes` controls how the vector space assists in query pruning. The more probes, the more accurate the search, but also the slower it is.
+- Type: integer
+- Default: ` `
+- Example:
+    - `SET vchordrq.probes = 1` means that only one probe is used.
+    - `SET vchordrq.probes = 10` means that ten probes are used.
+- Note: The default value is ` `, which means that only one probe is used. This is the fastest search, but also the least accurate. If you want to improve the accuracy of the search, you can increase the number of probes. However, this will also slow down the search.
+
+#### `vchordrq.epsilon`
+    
+- Description: Even after pruning, the number of retrieved vectors remains substantial. The index employs the RaBitQ algorithm to convert vectors into bit vectors, which require just $\frac{1}{32}$ the memory of single-precision floating-point vectors. With minimal floating-point operations, most computations are integer-based, leading to faster processing. Unlike typical quantization algorithms, RaBitQ not only estimates distances but also their lower bounds. The index computes the lower bound for each vector and dynamically adjusts the number of vectors needing recalculated distances, based on the query count, thus balancing performance and accuracy. The GUC parameter `vchordrq.epsilon` controls the conservativeness of the lower bounds of distances. The higher the value, the higher the accuracy, but the worse the performance. The default value provides unnecessarily high accuracy for most indexes, so you can try lowering this parameter to achieve better performance.
+- Type: float
+- Default: `1.9`
+- Example:
+    - `SET vchordrq.epsilon = 0.1` indicates that high accuracy is not required.  
+    - `SET vchordrq.epsilon = 4.0` means you need a very high accuracy.
```

Comment:
```suggestion
    - `SET vchordrq.epsilon = 0.1` indicates you are using a very optimistic lower bound estimation. You set it this way because your dataset is not sensitive to the lower bound estimation, for the precision you need.
    - `SET vchordrq.epsilon = 4.0` indicates you are using a very pessimistic lower bound estimation. You set it this way because your dataset is not very sensitive to the lower bound estimation, for the precision you need.
```